src/main/java/sirius/biz/importer/AutoImport.java

```diff
@@ -28,4 +36,20 @@
      * @return the list of permissions required to be able to import the annotated field
      */
     String[] permissions() default {};
+
+    /**
+     * Hides the field in the job documentation
+     *
+     * @return whether the field is hidden in the job documentation
+     */
+    boolean hidden() default false;
+
+    /**
+     * Marks the field as required in the job documentation.
+     * <p>
+     * In most cases, the framework logic will work this out by itself. Use this flag if it doesn't.
+     *
+     * @return true if the field is required in the import
+     */
+    RequiredStatus value() default RequiredStatus.AUTO_DETECT;
```

Comment:
Optional: name this required() or the like?

Reply:
Advantage of `value` is that you can write `@AutoImport(RequiredStatus.OPTIONAL)` which reads kind of nicely compared to `@AutoImport(required = RequiredStatus.OPTIONAL)`